calico/felix/endpoint.py

```diff
@@ -81,6 +103,84 @@ def _create(self, combined_id):
                              self.fip_manager,
                              self.status_reporter)
 
+    @actor_message()
+    def on_tier_data_update(self, tier, data):
+        """
+        Message received when the metadata for a profile tier is updated
+        in etcd.
+
+        :param str tier: The name of the tier.
+        :param dict|NoneType data: The dict or None, for a deletion.
+        """
+        _log.debug("Data for security tier %s updated to %s", tier, data)
```

Comment:
Oh dear, another terminology change.  It really would help if the code and comments could use consistent naming.

I suspect the underlying problem here is that we haven't yet written the public documentation for this feature.